util/demo.js

```diff
@@ -212,15 +211,15 @@ module.exports = async function (args, context = epochtal) {
 
       let ppnf = false, sv_cheats = false;
       let lastTimestamp = null, speedrunTimer = null;
-      let timescale = [], timescaleTotal = 0;
+      const serverTimestamp = Date.now();
```

Comment:
this _reeeally_ should be something like `referenceTimestamp`, or pretty much anything that would resolve the ambiguity of referring to two different things with the name "server timestamp"